pkg/crc/machine/start.go

```diff
@@ -164,51 +169,52 @@ func runGrowpart(sshRunner *crcssh.Runner, rootPart string) error {
 	return nil
 }
 
-func growLVForMicroshift(sshRunner *crcssh.Runner) error {
+func growLVForMicroshift(sshRunner *crcssh.Runner) (string, error) {
 	rootPart, _, err := sshRunner.RunPrivileged("Get PV disk path", "/usr/sbin/pvs", "-S", "lv_name=root", "-o", "pv_name", "--noheadings")
 	if err != nil {
-		return err
+		return "", err
 	}
 	rootPart = strings.TrimSpace(rootPart)
 	if err := runGrowpart(sshRunner, rootPart); err != nil {
-		return err
+		return "", err
 	}
 
 	if _, _, err := sshRunner.RunPrivileged("Resizing the physical volume(PV)", "/usr/sbin/pvresize", rootPart); err != nil {
-		return err
+		return "", err
 	}
 
 	// Get the size of volume group
 	sizeVG, _, err := sshRunner.RunPrivileged("Get the volume group size", "/usr/sbin/vgs", "--noheadings", "--nosuffix", "--units", "b", "-o", "vg_size")
 	if err != nil {
-		return err
+		return "", err
 	}
 	vgSize, err := strconv.Atoi(strings.TrimSpace(sizeVG))
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	// Get the size of root lv
 	sizeLV, _, err := sshRunner.RunPrivileged("Get the size of root logical volume", "/usr/sbin/lvs", "-S", "lv_name=root", "--noheadings", "--nosuffix", "--units", "b", "-o", "lv_size")
 	if err != nil {
-		return err
+		return "", err
 	}
 	lvSize, err := strconv.Atoi(strings.TrimSpace(sizeLV))
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	// vgFree space as part of the bundle is default to ~15.02G (16127098880 byte)
 	vgFree := 16127098880
 	expectedLVSize := vgSize - vgFree
 	sizeToIncrease := expectedLVSize - lvSize
+	lgVol := "/dev/rhel/root"
```

Comment:
Instead of hardcoding `/dev/rhel/root` here and returning it back to the caller, I'd pass `rhel/root` as a `lvFullName` argument, and make this method independent of the VG/PV used (lgVol becomes `dev`+`lvFullName`, and the various `-S lv_name=root` can be replaced with `-S lv_full_name=rhel/root`).